### ISSUE TITLE
clamav freshclam hitting CDN rate limits

### ISSUE THREAD

**Issue by maxadamo** [mikenye/docker-postfix#30]
I am trying to setup this container for the first time. 
I got an issue with Nomad, for some reason freshclam failed a few times, and I got banned for 1 hour with the following message:

```
2024-05-01 11:14:01.024682668  [freshclam]  3. If you have more than 10 hosts on your network attempting to download,
2024-05-01 11:14:01.025815402  [freshclam]     it is recommended that you set up a private mirror on your network using
2024-05-01 11:14:01.025819441  [freshclam]     cvdupdate (https://pypi.org/project/cvdupdate/) to save bandwidth on the
2024-05-01 11:14:01.025820913  [freshclam]     CDN and your own network.
```

Unfortunately the mirror is hardcoded here: 
https://github.com/mikenye/docker-postfix/blob/master/rootfs/etc/cont-init.d/30-clamav-generate-freshclam.conf.sh#L13

Imagine I have multiple images, on multiple environment. I thought it's worth setting up this mirror, but how do I tell Clamav, on this container, to use my mirror?

**Comment by maxadamo** [comment]
on another note, in my understanding `/var/lib/clamav` can be shared among container, and it will reduce the number of requests. 
I'll share my Terraform/Nomad job once I get it working.

**Comment by maxadamo** [comment]
fixed by #31
